2017-2018/ROS_RoboBuggy/src/robobuggy/include/transistor_serial_messages.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _TRANSISTOR_SERIAL_MESSAGES_H_
```

Comment:
I would actually prefer if we did something like low level, where the header file is generated at compile time. Is there a way we can do that?

Reply:
Talked with Vivek and we decided that this is far too much effort for not enough gain. For now we're going to maintain our own copy and hope that they sync up properly